pyproject.toml

```diff
@@ -3,5 +3,5 @@ requires = [
     'setuptools>=65.5.1',
     'wheel',
     'numpy>=1.22.0',
-    'Cython',
+    'Cython>=0.29.0,<3.0.0',
```

Comment:
oh this is related to compiling error of geowombat 👍

Reply:
> I understand these changes address these problem: https://github.com/jgrss/cultionet/actions/runs/4323904833/jobs/7548059059
Yes! I ran tests locally and everything passed.

> Will CI run the test in this branch or will it only run when you merge it into jgrss/topo_v2_time_rnn_nobal_batch_act_dgm_test?
Yeah, I guess it's not triggered here.